policyengine_us/parameters/gov/irs/payroll/disability_income_exclusion/cap.yaml

```diff
@@ -0,0 +1,14 @@
+description: IRS caps the disability exclusion to this amount per disabled person.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Disability exclusion cap
+  reference:
+    - title: 26 U.S. Code § 105 (d)(3) 
+      href: https://tile.loc.gov/storage-services/service/ll/uscode/uscode1982-01002/uscode1982-010026001/uscode1982-010026001.pdf#page=209
+    # 26 U.S. Code § 105 (d)(3) was repealed in 1983, but some states still uses it as a standard when computing disability income exclusion 
+    - title: Code of the District of Columbia § 47–1803.02. (a)(2)(M)
```

Comment:
Remove this citation, as it does not concern the values here. We don't cite pointers. Only cite the DC rule in the variable.